<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundWidget</class>
 <widget class="QMainWindow" name="SoundWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="PlotDisplayAreaWidget" name="plotDisplayWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionData_Manager"/>
    <addaction name="actionPlot_Manager"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRegressions">
    <property name="title">
     <string>Regressions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnnotations">
    <property name="title">
     <string>Annotations</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuScripting">
    <property name="title">
     <string>Scripting</string>
    </property>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuRegressions"/>
   <addaction name="menuAnnotations"/>
   <addaction name="menuScripting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData_Manager">
   <property name="text">
    <string>Data Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPlot_Manager">
   <property name="text">
    <string>Plot Manager</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotDisplayAreaWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotdisplayareawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
